web/src/stream_popover.js

```diff
@@ -214,7 +214,29 @@ function build_stream_popover(opts) {
     });
 }
 
-export function build_move_topic_to_stream_popover(
+async function get_message_type(current_stream_id, topic_name, current_message_id) {
+    const lastMessageId = await new Promise((resolve) => {
+        message_edit.with_last_message_id(current_stream_id, topic_name, (message_id) => {
+            resolve(message_id);
+        });
+    });
+
+    const firstMessageId = await new Promise((resolve) => {
+        message_edit.with_first_message_id(current_stream_id, topic_name, (message_id) => {
+            resolve(message_id);
+        });
+    });
```

Comment:
Remember that we don't use `camelCase` in variable names.

Reply:
Fixed and the variables are now renamed to `is_oldest` and `is_newest`.